master-detail-product/README.md

```diff
@@ -4,11 +4,13 @@ A utility process with one master dialog(with detail view) and a stand-alone det
 
 ## Demo
 
-In this demo, a lazy datatable is listing all the products available in the Ivy business repo.
-There exist three scenarios:
-1. User adds a new product (demonstrated by modal dialog opened in the same HTML dialog as the master).
-2. User edits an existing product (demonstrated by process design: user is redirected to the stand-alone detail dialog and when finished with editing returns to the master dialog).
-3. User edits an existing product (demonstrated by a detail view contained in the master dialog and navigating by primefaces).
+Axon Ivy's Master-detail interface best-practice example is designed to accelerate process automation initiatives by demonstrating the flexible and powerful UI possibilities. This example showcases the following features:
+* Three different approaches to implementing the Master-Detail Pattern are demonstrated based on the project's complexity. These approaches involve utilizing different process elements, UI views, and modal dialogs.
+
```

Comment:
additional line-breaks: as on line 9 and 11 are not real markdown. but instead a break should be added before the list (so after : on line 7)
you may use a markdown editor, such as http://pad.haroopress.com/user.html , to help you with it.